data/json/items/tools.json

```diff
@@ -5401,6 +5410,7 @@
     "symbol": "/",
     "color": "dark_gray",
     "use_action": "PICKAXE",
+    "techniques" : [ "WBLOCK_1" ],
```

Comment:
Inappropriate space here `"techniques" :`, should be `"techniques":`

I know some jsons aren't consistent about this (most likely haven't been formatted).